dropwizard-metrics-datadog/src/main/java/io/dropwizard/metrics/DatadogReporterFactory.java

```diff
@@ -34,5 +39,14 @@ public ScheduledReporter build(MetricRegistry registry) {
         .convertDurationsTo(getDurationUnit())
         .convertRatesTo(getRateUnit())
         .build();
-    }
+  }
+
+  @Override
+  public MetricFilter getFilter() {
```

Comment:
I take back what I said about `useRegexFilters()` since it appears you're using it mainly as a getter for testing purposes.